src/components/v2v3/shared/SplitItem.tsx

```diff
@@ -213,6 +257,8 @@ export default function SplitItem({
               valueSuffix={valueSuffix}
               valueFormatProps={valueFormatProps}
               currency={currency}
+              showFees={showFees}
+              isJuiceboxProject={isJuiceboxProject}
```

Comment:
**suggestion** rather than making this a prop, i think we should:

1. Extract the `isJuiceboxProjectSplit` code into a helper function.
2. Use it in this file on line 213
3. Use it within the `SplitItem.tsx` component